README.md

```diff
@@ -32,47 +32,52 @@ A hotel booking application in React. Homework for the [CodeYourFuture React mod
 
-6. Instead of using your hard-coded data in the `<SearchResults />` component, load data from the `src/data/fakeBookings.json` file in the `<Bookings />` component and pass it as a prop to `<SearchResults />`. Hint: look in the `<Bookings />` component for how to import data from a JSON file.
+6. Instead of using your hard-coded data in the `<SearchResults />` component, load data from the `src/data/fakeBookings.json` file in the `<Bookings />` component and pass it as a prop to `<SearchResults />`. All the bookings in `src/data/fakeBookings.json` should now be displayed in your table. Hint: look in the `<Bookings />` component for how to import data from a JSON file.
 
-7. Add another column to your `<SearchResults />` table which shows the number of days each booking is staying. Hint: try installing the [moment.js library](http://momentjs.com/) (you'll need to install it with `npm install moment --save`) and using the [`.diff()` method](http://momentjs.com/docs/#/displaying/difference/) to compare dates
+7. Add another column to your `<SearchResults />` table which shows the number of nights a guest is staying. For example, Mr John Doe has a booking for **2** nights. Hint: try installing the [moment.js library](http://momentjs.com/) (you'll need to install it with `npm install moment --save`) and using the [`.diff()` method](http://momentjs.com/docs/#/displaying/difference/) to compare dates.
 
 ### Lesson 2
 
-8. Within `src/App.js`, render the `<Restaurant />` component (that is provided for you in `src/Restaurant.js`) underneath the `<Bookings />` component. Then convert the `<Restaurant />` component to a class component.
+8. Within `src/App.js`, render the `<Restaurant />` component (that is provided for you in `src/Restaurant.js`) underneath the `<Bookings />` component.
 
-9. Add a method to the `Restaurant` class named `addOrder`. Use `console.log` to log a "Add order" message. Remember to use the `addOrder = () => {}` syntax. Add a `onClick` handler to the Add `<button>` that calls `this.addOrder`. Ensure that clicking on the button logs your "Add order" message in the console.
+9. At the moment, the number of pizzas a guest can order is static and set to 0, even if they click on the 'Add' button. We will change that in the following to let a guest add more pizzas to their order. First, declare a new state variable `orders` along the function to set the orders state `setOrders`. The initial value of the `orders` state should be **0**. Use the new `orders` variable instead of the `pizzas` variable (that you can now delete) and verify the number of ordered pizzas it still **0** on the screen. Hint: you need to use the React function `useState` to create a state variable. Remember to import the function at the top with `import React, {useState} from "react";`.
```

Comment:
```suggestion
9. At the moment, the number of pizzas a guest can order is static and set to 0, even if they click on the 'Add' button. We will change that in the following to let a guest add more pizzas to their order. First, declare a new state variable `orders` along with the function to set the orders state `setOrders`. The initial value of the `orders` state should be **0**. Use the new `orders` variable instead of the `pizzas` variable (that you can now delete) and verify the number of ordered pizzas it still **0** on the screen. Hint: you need to use the React function `useState` to create a state variable. Remember to import the function at the top with `import React, {useState} from "react";`.
```

Reply:
Would it be worth adding a link to docs for useState? Or will that be too much of a hint? https://reactjs.org/docs/hooks-state.html